package.json

```
@@ -11,7 +11,8 @@
     "start": "next start",
     "lint": "next lint",
     "format": "prettier --write .",
-    "prisma:migrate": "prisma migrate dev --name",
+    "prisma:create": "prisma migrate dev --name",
+    "prisma:apply": "prisma migrate dev",
```

Creo que se me ocurrió un mejor nombre para los comandos, qué opinas de estos?

```suggestion
    "create-migration": "prisma migrate dev --name",
    "apply-migrations": "prisma migrate dev",
```